src/test/java/io/appium/java_client/ios/XCUIAutomationTest.java

```diff
@@ -91,4 +92,14 @@
         driver.rotate(landscapeLeftRotation);
         assertEquals(driver.rotation(), landscapeLeftRotation);
     }
+
+    @Test public void testTouchId() {
+        try {
+            driver.touchId(true);
+            driver.touchId(false);
+            assertEquals(true, true);
+        } catch(Exception e){
+            assertEquals(true, false);
```

Comment:
@dpgraham I think it is more senseful to throw an exception.
Otherwice we have a risk to produce and invalid build or to not react to server changes.